pages/linux/photorec.md

```diff
@@ -0,0 +1,12 @@
+# PhotoRec
+
+> PhotoRec is file data recovery software designed to recover lost files including video, documents and archives from hard disks, CD-ROMs, and lost pictures (thus the Photo Recovery name) from digital camera memory.
+> Always write file recovered with PhotoRec to another drive than the source drive.  https://www.cgsecurity.org/wiki/PhotoRec.
```

Comment:
Again, these lines look like they are from the man page. We generally try to be more concise thant he man page. What about this instead of both lines?
```
> Deleted file recovery tool.
> It is recommended to write recovered files to a disk separate to the one being recovered from.
> More information: <https://www.cgsecurity.org/wiki/PhotoRec>.
```